draft-ietf-ppm-dap.md

```diff
@@ -146,7 +146,7 @@ Input:
 
 Input share:
 : An aggregator's share of the output of the VDAF
-   {{?VDAF=I-D.draft-irtf-cfrg-vdaf-01}} sharding algorithm. This algorithm is run by
+   {{?VDAF=I-D.draft-irtf-cfrg-vdaf-03}} sharding algorithm. This algorithm is run by
```

Comment:
While we're in here touching these lines: I think VDAF qualifies as a normative reference rather than an informative one based on [the IETF's criteria](https://www.ietf.org/about/groups/iesg/statements/normative-informative-references/). This and all the other VDAF references should be updated to use `!`.
```suggestion
   {{!VDAF=I-D.draft-irtf-cfrg-vdaf-03}} sharding algorithm. This algorithm is run by
```

Reply:
Done!